Makefile

```diff
@@ -25,34 +28,58 @@ unlink:
 
 .PHONY: lint
 lint:
+	npm --prefix es-bs-base run lint
+# TODO: set up linting for es-ds-components and es-ds-docs
+# 	npm --prefix es-ds-components run lint
+# 	npm --prefix es-ds-docs run lint
+
+.PHONY: legacy-lint
+legacy-lint:
```

Comment:
I wonder if we should do this the other way around i.e. `lint` runs the `lint` command for all packages, and `lint-next` runs only prime-related packages. This would avoid having to change anything in regular CI/CD as we're still maintaining all packages ATM (i.e. `es-vue-base`)